### PR TITLE
Wm sw emailing

### PR DESCRIPTION
**Pull request by billyacademy**
We added email functionality and feature testing.

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 class ReviewsController < ApplicationController
   def create
     @playlist = Playlist.find(params[:playlist_id])
+    @user = @playlist.user.email
```
Comment (atsheehan) [inline app/controllers/reviews_controller.rb]: I would name this variable `@user_email`.

File: config/initializers/email.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+if Rails.env.development?
+  ActionMailer::Base.delivery_method = :smtp
+  ActionMailer::Base.smtp_settings = {
+    address: "localhost",
+    port: 1025
+  }
+end
```
Comment (atsheehan) [inline config/initializers/email.rb]: This is already defined in your development environment via `config/environments/development.rb`.

File: spec/mailers/user_mailer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+#     'friend@example.com', Time.now).deliver_now
+#     assert_not ActionMailer::Base.deliveries.empty?
+#
+#     # Test the body of the sent email contains what we expect it to
+#     assert_equal ['me@example.com'], email.from
+#     assert_equal ['friend@example.com'], email.to
+#     assert_equal 'You have been invited by me@example.com', email.subject
+#     assert_equal read_fixture('invite').join, email.body.to_s
+#   end
+# end
```
Comment (atsheehan) [inline spec/mailers/user_mailer_spec.rb]: Delete this file if it's not being used.

File: config/initializers/email.rb (hunk 2 above)
Comment (billyacademy) [inline config/initializers/email.rb]: We moved the code from development.rb to production.rb as that is where it belongs.

Comment (billyacademy) [inline config/initializers/email.rb]: Or so we think..

**Pull request merged**